Competition-2020/src/main/include/Robot.h

```diff
@@ -15,9 +15,30 @@
 
 using namespace frc;
 
+class XboxController2 : public XboxController {
```

Comment:
This should be in a separate file

Reply:
Another comment on the whole file... since we've got a larger team, might as well have better coding. Please add comments that include return values, parameters, and a short summary. Don't worry too much about how it is styled right now, we'll do that later.